src/analysis/problematic_exclusions.py

```diff
@@ -31,13 +31,12 @@
 OUTDIR = REPORTS_DIR
 SPARQL_STR_GET_LABELS = """
 {% for sparql_prefix_line in prefixes %}{{ sparql_prefix_line }}{% endfor %}
-prefix owl:  <http://www.w3.org/2002/07/owl#>
-prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX owl:  <http://www.w3.org/2002/07/owl#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 
-select ?term_id ?term_label where {
+SELECT ?term_id ?term_label WHERE {
     VALUES ?term_id { {{values}} }
-    ?term_id a owl:Class;
-      rdfs:label ?term_label .
+    ?term_id rdfs:label ?term_label .
```

Comment:
## Bug fix 1
I'm not sure why this is; maybe @matentzn knows, but there are some terms (at least this 1 term `icd11.foundation:430573082`) that are `owl:Class` in the `component-download`, but when that file gets converted to the processed component (in the `components/` dir, it no longer appears in a class, but there is a `rdf:Description` for it.

<details><summary>tmp/component-download-icd11foundation.owl.owl vs components/icd11foundation.owl</summary>
<p>
`tmp/component-download-icd11foundation.owl.owl`:

```owl
    <owl:Class rdf:about="http://id.who.int/icd/entity/430573082">
        <rdfs:subClassOf rdf:resource="http://id.who.int/icd/entity/1029961628"/>
        <rdfs:subClassOf rdf:resource="http://id.who.int/icd/entity/1661422782"/>
        <schema:browserUrl rdf:resource="https://icd.who.int/dev11/f/en#/http%3A%2F%2Fid.who.int%2Ficd%2Fentity%2F430573082"/>
        <schema:inclusion xml:lang="en">Adult T-cell lymphoma</schema:inclusion>
        <skos:altLabel xml:lang="en">Adult T-cell leukaemia</skos:altLabel>
        <skos:altLabel xml:lang="en">Adult T-cell lymphoma</skos:altLabel>
        <skos:altLabel xml:lang="en">Adult T-cell lymphoma/leukaemia</skos:altLabel>
        <skos:prefLabel xml:lang="en">Adult T-cell leukaemia/lymphoma (HTLV-1 positive)</skos:prefLabel>
    </owl:Class>
```

`components/icd11foundation.owl`:
```owl
<rdf:Description rdf:about="http://id.who.int/icd/entity/430573082">
        <oboInOwl:hasExactSynonym xml:lang="en">Adult T-cell leukaemia</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym xml:lang="en">Adult T-cell leukaemia/lymphoma (HTLV-1 positive)</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym xml:lang="en">Adult T-cell lymphoma</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym xml:lang="en">Adult T-cell lymphoma/leukaemia</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym>adult t-cell leukaemia/lymphoma</oboInOwl:hasExactSynonym>
        <rdfs:label xml:lang="en">Adult T-cell leukaemia/lymphoma (HTLV-1 positive)</rdfs:label>
    </rdf:Description>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://id.who.int/icd/entity/430573082"/>
        <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasExactSynonym"/>
        <owl:annotatedTarget>adult t-cell leukaemia/lymphoma</owl:annotatedTarget>
        <oboInOwl:hasSynonymType rdf:resource="http://purl.obolibrary.org/obo/mondo#GENERATED"/>
    </owl:Axiom>
```

</p>
</details> 

In order to fix this problem, I made the SPARQL query less strict, dropping the `owl:Class` requirement.

The results are that this fixes the ICD11 issue we're getting, but otherwise everything else is unaffected.

Reply:
I don't agree with this fix - it's should be totally ineffectual; if this is needed, we have a much bigger problem.. are you saying the ICD11 ingest owl has no rdf:type owl:Class assertions? If so, we should load safe it with robot once before processing. 

Minor comment: I would suggest always to avoid spurious diffs; this has obviously been cycled to some AI tool, and out of the 4 changes in this file only 1 matters. Same with the Python script below..

Reply:
The "tmp" file uses `owl:Class` and the "component" file uses `rdf:Description` for some entities and `icd11.foundation:430573082` is one of those that changes between the two files.

Reply:
😱 looking into this now

Reply:
Found it..

```
		remove -T $(TMPDIR)/icd11foundation_relevant_signature.txt --select complement --select "classes individuals" --trim false \
```

(problem is --trim false). Drilling deeper.

Reply:
Ok. 

Removing --trim false from the line in the comment above will sort this issue, and no need to change the SPARQL query. This is not related to the other issue (the question of wether to use the mirror or the component).

The disease you named (430573082 ) is an extension code and therefore _not_ in the list of entities of interest.

Reply:
> The disease you named (430573082) is an extension code and therefore not in the list of entities of interest.

I suppose that is why it is an exclusion. The point of the script is to report on excluded terms that have mondo xrefs. So it makes sense that this disease is appears here.
Are you mentioning this just as an FYI? Or is there something wrong that needs to be changed?

Reply:
> avoid spurious diffs; this has obviously been cycled to some AI tool

@matentzn No, that's not the case; these are all intentional, manual changes. I can explain any/each of these changes if you like. I've heard from you before that basically I don't need a separate PR for codestyle changes that are closely related to what I am working on in the PR. If I noticed codestyle issues in something that was not "problematic exclusions", I would put them in a separate PR.

The only reason I didn't explain why I made these changes is because:
- I'm trying to write less overall / keep to what is important/necessary.
- It was suggested that I don't need to comment about some obvious things because they can clearly be seen in the diff.